.travis.yml

```diff
@@ -3,8 +3,37 @@ language: python
 python:
   - "2.7"
   - "3.5"
+  - "3.6"
 install:
   - pip install -r requirements.txt
-script: 
-  - pylint adal
-  - python -m unittest discover -s tests 
+script:
+  - # PyLint does not yet support Python 3.6 https://github.com/PyCQA/pylint/issues/1241
+    if [ "$TRAVIS_PYTHON_VERSION" != "3.6" ]; then
+      pylint adal;
+    fi
+  - python -m unittest discover -s tests
+
+deploy:
+  - # test pypi
+    provider: pypi
+    distributions: sdist
+    server: https://testpypi.python.org/pypi
+    user: "nugetaad"
+    password:
+      secure: sEaIkr4od/4NJJq/3nvxjC/K/MF/KGBmN148dB4Sj/xMCA3DqqESMxhA8pB4YWfeZHHC6SOTqNHs6SEOxXg5JfY3eOoAke2GV0z/5VaDu/pYbjBZctxi2HC23hcQhDglW1nwfIw8Oglr9VLMbY8lls5AM69E4L/S8vB/hZvwl85WGsLIpod186GT00Ni0W+wnPVNJK+XP7xVb1BM1SRfioVDIsQ9W4qFWEYOLjJQvZByJvkb6Q971rwewZJFihYOZ3ynNeu+3lxifuCqag5KJgRmvaEynMuXx7NhO8kTic+kgcSG2YyDKBWVCrq5/8jJgAAj21n1QuyU3sdPtpVA/yoL41Lr2j1/lIsLbwzJdgaYcwn5QY8VccBNQyGctO/nSvDX6SbFnC1mYUb8J48AFg/EUsTmwqWqkXfBYwSRqgIqeMhh9vGYNh8Fxe9rRaAYTNMQa8ZZ6vBMoO71NhV4WC4tJEJ2KruWqrO9hUFHs4xV0EvAes9uCyzwk/pPE3oCZXt3BlKN5VMlaWzhA06g8oIIuovNnAKeqGXFKRwcVqVoPq+XWYfBy6OLP5b2rTBaYmTMNTkKEbTup66mP/DYbSN/j4g2wuW8koagg1p7a5I+lXrwBaDzLPBeJOxPCdzaU9rZJI8dsvFLDKqUY/wcbE8SoIMLwSwsudzMB5nS1Hw=
```

Comment:
Very good question. It is an encrypted value only useful for Travis-CI only. No 3rd party can decrypt and use it. [Travis provides this feature](https://docs.travis-ci.com/user/encryption-keys/) to so that we don't need (and we won't) store plain text here.